app/wallet/wallet.go

```diff
@@ -0,0 +1,212 @@
+package wallet
+
+import (
+	"database/sql"
+	"errors"
+	"fmt"
+
+	ljsonrpc "github.com/lbryio/lbry.go/v2/extras/jsonrpc"
+	"github.com/lbryio/lbrytv/app/sdkrouter"
+	"github.com/lbryio/lbrytv/internal/lbrynet"
+	"github.com/lbryio/lbrytv/internal/monitor"
+	"github.com/lbryio/lbrytv/models"
+	"github.com/volatiletech/sqlboiler/queries/qm"
+
+	"github.com/lib/pq"
+	pkgerrors "github.com/pkg/errors"
+	"github.com/sirupsen/logrus"
+	"github.com/volatiletech/sqlboiler/boil"
+)
+
+var logger = monitor.NewModuleLogger("wallet")
+
+func DisableLogger() { logger.Disable() } // for testing
+
+// TokenHeader is the name of HTTP header which is supplied by client and should contain internal-api auth_token.
+const TokenHeader = "X-Lbry-Auth-Token"
+const pgUniqueConstraintViolation = "23505"
+
+// Retrieve gets user by internal-apis auth token. If the user does not have a wallet yet, they
+// are assigned an SDK and a wallet is created for them on that SDK.
+func GetUserWithWallet(rt *sdkrouter.Router, internalAPIHost, token, metaRemoteIP string) (*models.User, error) {
```

Comment:
We are not returning wallets from this function, in fact, they are not mentioned anywhere in the function code, so the name is a bit misleading.